rcl/src/rcl/graph.c

```diff
@@ -667,6 +672,45 @@ rcl_wait_for_subscribers(
     rcl_count_subscribers);
 }
 
+
+rcl_ret_t
+rcl_wait_for_clients(
```

Comment:
Those are not exactly related to the service info verbose feature, but nice to have, and actually used in the test.